CONTRIBUTING.md

```diff
@@ -1,6 +1,46 @@
 # Contributing to Fn
 
-We welcome all contributions!
+We welcome all contributions! We accept contributions through GitHub pull
+requests and issues. Before setting out on making any significant
+contribution, please file an issue or assign an issue to yourself.
+
+## Reporting security issues
+
+The Fn maintainers take security seriously. If you discover a security
+issue, please bring it to their attention right away!
+
+Please **DO NOT** file a public issue, instead send your report privately to
+[TODO@fnproject.io](mailto:TODO@fnproject.io). Please have as the
```

Comment:
@carimura @tteggel i know we own fnproject.io, we could set an MX for security@fnproject.io to point to one of our individual email accounts for now, or set up a mailing list so that a few of us get this. up to us, need something but doesn't have to be much.

Reply:
I have requested a mailing list on the corporate systems. Will take a day or two to approve and propagate.

Reply:
can help, let me know. happy to setup security@

Reply:
bump @tteggel did we get an email for this?

Reply:
security@fnproject.io has been setup and currently goes to me and @rdallman. Can change to a wider core team distro later.